egs/madcat_ar/v1/local/apply_bpe.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Rico Sennrich
```

Comment:
this code, from their repo, seems to be released under the MIT license.  You can add
`# Released under the MIT License.`